source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -157,6 +181,39 @@ content: |
            npm package. See the `npmjs documentation on the uuid-base64
            package <https://www.npmjs.com/package/uuid-base64>`_ for
            installation instructions.
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
```

Comment:
Could this also highlight important lines similar to the other two driver examples?

Reply:
Fixed:
- Added highlighting similar to other drivers